NGitLab/Models/TestReportSummary.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Runtime.Serialization;
+
+namespace NGitLab.Models
+{
+    [DataContract(Name = "total")]
```

Comment:
I don't think that will work.

`TestReport` and `TestReportSummary` does share 'test_suites', but other props are not the same.

If you are doing that, I would bet that you will have the 'total' property empty (cause of different names) and no 'test_suites' (cause it will not find any under 'total' prop)

I don't think `TestReportSummary` will be able to share much with `TestReport` except the `TestSuites` prop.

Reply:
If the 2 methods return different things, then create 2 different classes.

Reply:
Fixed.